starter-code/scripts/views/articleView.js

```diff
@@ -37,55 +38,57 @@
   };
 
   // COMMENT: What does this method do?  What is it's execution path?
+  // This method has an event listener on the filters dropdown for both author and category. Once something is selected, it uses the value captured by the event to fire off the functionality to replace the element with that id.
```

Comment:
This method is an event listener that only allows an event to trigger the first time something is selected. Once a select event happens in our filters, we save a reference to the resource, whether it be an author or category (by targeting the css id selector and removing the suffix -filter [replacing it with an empty string, essentially nothing] so we're left with "author" or "category" -- this way we can deal with both filters in one method). It then "selects" the title option from the other select box (as that title has a value of '') so that it reverts back to the original look, on the title text (as it wasn't being used to select the current filter).

Finally, we redirect our client to a route that we've laid out with Page.js, either /author/:authorName or /category/:categoryName, replacing any spaces with + signs in our URL.